documentation/book/proc-manual-restart-zookeeper.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='proc-manual-restart-zookeeper-{context}']
+= Manually restart a Zookeeper cluster
+
+This procedure describes how to manually restart an existing Zookeeper cluster by using an {ProductPlatformName} annotation.
+
+.Prerequisites
+
+* A running Zookeeper cluster.
+* A running Cluster Operator.
+
+.Procedure
+
+. Find the name of `StatefulSet` which controls Zookeeper pods to restart.
++
+Lets say the cluster is named _my-cluster_. The name of the Zookeeper StatefulSet is _my-cluster-zookeeper_.
++
+
+. Annotate `StatefulSet` resource in {ProductPlatformName}.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl annotate statefulset _cluster-name_-zookeeper cluster.operator.strimzi.io/manual-rolling-update=true
```

Comment:
the annotation is without "cluser." now

Reply:
Can you show  me the PR which changes `io.strimzi.operator.cluster.ClusterOperator#STRIMZI_CLUSTER_OPERATOR_DOMAIN` ? Because using annotation without `cluster.` does not work.